### ISSUE TITLE
Vue warning after upgrading to 0.1.3

### ISSUE THREAD

**Issue by ledermann** [inertiajs/inertia-vue#132]
After upgrading to **0.1.3** I got this warning in the browser console in my application:

```
[Vue warn]: Invalid default value for prop "only": Props with type Object/Array must use 
a factory function to return the default value.
```

Seems to be caused by #131. I'm using Vue v2.6.11

**Comment by Juhlinus** [comment]
Nice catch @ledermann !

I'll work on a PR.

**Comment by sebastiandedeyne** [comment]
Sorry about this, fixed in `0.1.4`!
